### PR TITLE
Gas Type Added in AbiItem

### PR DESCRIPTION
**Pull request by maektwain**
As suggested by the #3383

## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran ```npm run dtslint``` with success and extended the tests and types if necessary.
- [x] I ran ```npm run test:unit``` with success.
- [ ] I have executed ``npm run test:cov`` and my test cases do cover all lines and branches of the added code.
- [ ] I ran ```npm run build-all``` and tested the resulting file/'s from  ```dist``` folder in a browser.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the live network.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -->
 
+## [1.2.6]
+Added gas type in Contract AbiItem
+
```
Comment (holgerd77) [inline CHANGELOG.md]: Please have a closer look at this file again! 😄

Comment (cgewecke) [reply CHANGELOG.md]: @maektwain Apologies, could you change this so it's only a single entry in the existing `1.2.7` section of the changelog, at the bottom of the file?

**Review by cgewecke — COMMENTED** [review]
This LGTM - there's a small issue with the changelog that still needs to be resolved - otherwise good to go.

File: CHANGELOG.md (hunk 1 above)
Comment (cgewecke) [reply CHANGELOG.md]: Made this change via Github UI, resolving....

**Review by cgewecke — APPROVED** [review]

**Comment by holgerd77** [comment]
Ok, will merge here.

**Pull request merged**